tutorials/backend/hasura-authentication/tutorial-site/content/integrations/auth0.md

```diff
@@ -163,81 +212,75 @@ It's time to integrate the newly created Auth0 application with Hasura.
 After adding the `admin secret`, you need to configure the public keys for Auth0. One way to generate the JWT config is
 to use the [Hasura JWT configurator](https://hasura.io/jwt-config/).
 
-![Hasura JWT Configurator](https://graphql-engine-cdn.hasura.io/learn-hasura/assets/graphql-hasura-authentication/auth0/hasura-jwt-config.png)
+![Hasura JWT Configurator](https://graphql-engine-cdn.hasura.io/learn-hasura/assets/graphql-hasura-authentication/auth0/auth0-images-2/auth0-hasura-generate-jwt-config.png)
 
-Choose the provider (Auth0) and then enter your "Auth0 Domain Name". After that, press the button "Generate Config" to
-get your JWT Config.
+Choose the provider (Auth0) and then enter your "Auth0 Domain Name" which you can find in your Auth0 application. After
+that, press the button "Generate Config" to get your JWT Config.
 
 You will set the value as an environment variable in your Hasura Project. To do so, go to the
 [Hasura Dashboard](https://cloud.hasura.io?skip_onboarding=true/) and then click the "Gear ⚙️" icon.
 
-![Hasura Project Env Vars](https://graphql-engine-cdn.hasura.io/learn-hasura/assets/graphql-hasura-authentication/auth0/hasura-project-env-vars.png)
-
 After that, go to the `Env vars` section and click the `+ New Env Var` option. See the image for reference.
 
+![Hasura Project Env Vars](https://graphql-engine-cdn.hasura.io/learn-hasura/assets/graphql-hasura-authentication/auth0/auth0-images-2/auth0-hasura-project-env-vars.png)
+
 Select `HASURA_GRAPHQL_JWT_SECRET` for the "Key" and then paste the JWT config generated previously.
 
-![Hasura JWT Secret](https://graphql-engine-cdn.hasura.io/learn-hasura/assets/graphql-hasura-authentication/auth0/hasura-jwt-secret.png)
+![Hasura JWT Secret](https://graphql-engine-cdn.hasura.io/learn-hasura/assets/graphql-hasura-authentication/auth0/auth0-images-2/auth0-hasura-project-add-new-env-var.png)
 
 Click on `Add`, and the environment variable will be applied to your project.
 
 Congratulations! Auth0 is successfully integrated with your Hasura instance.
 
-## Test Database Syncing Rule
+## Test Users Syncing Action
 
 Go to your Hasura app dashboard and create a `users` table with the following columns:
```

Comment:
Not a blocker, but given that `app` and `dashboard` are used throughout Auth0, it may be clearer to refer to this as `Console` as we do everywhere else.